inspektor/lint.py

```diff
@@ -64,6 +64,7 @@ def get_opts(self):
             if self.args.enable:
                 pylint_args.append('--enable=%s' % self.enabled_errors)
             pylint_args.append('--reports=no')
+            pylint_args.append('--errors-only')
```

Comment:
If you are talking about the https://github.com/avocado-framework/avocado/pull/2037 I even tried the custom report style and did not succeeded 😞. I'd like to know where is it coming from (I can't reproduce it locally)